components/layout/display_list/mod.rs

```diff
@@ -852,6 +897,18 @@ impl Fragment {
             }
         }
 
+        let lcp_candidate_id = fragment
```

Comment:
This is incorrect.

For text records, we don't create an LCP candidate for each text fragment. We create an entry for the containing block of text.

See:
- Step 2.4 for https://www.w3.org/TR/largest-contentful-paint/#sec-report-largest-contentful-paint
- Definition and calculation of [set of owned text nodes](https://w3c.github.io/paint-timing/#set-of-owned-text-nodes)